src/vm/excep.cpp

```diff
@@ -5532,6 +5527,22 @@ DefaultCatchHandlerExceptionMessageWorker(Thread* pThread,
         }
 
         PrintToStdErrA("\n");
+
+        // Send the log to Windows Event Log
+        EX_TRY
+        {
+            EventReporter reporter(EventReporter::ERT_UnhandledException);
+            if (!message.IsEmpty())
+            {
+                reporter.AddDescription(message);
```

Comment:
Although I think it is conceptually a bit cleaner to have the error reporting functionality within DefaultCatchHandler, there were several other cases and fallback paths in the old event logging code that are lost now. There is also an issue that the code InternalUnhandledExceptionFilter_Worker called DoReportForUnhandledException in a broader set of cases than it calls DefaultCatchHandler.